packages/jsActions/mobile-resources-native/package.json

```diff
@@ -11,12 +12,19 @@
     "minimumMXVersion": "9.0.5",
     "marketplaceId": 109513
   },
+  "docker": {
```

Comment:
If you do this than we cannot reuse this logic in widgets. Can we align and use `minimumMXVersion` instead?